app/src/components/CardGrid.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { AllAnimalsQuery } from "../graphql/generated";
+import { AnimalData, Animal } from "./Interfaces";
+
+// it is necessary to define the prop types here before they are recieved
+interface Props {
+  status: string;
+  isFetching: boolean;
+  data: AllAnimalsQuery | undefined;
+}
+
+const CardGrid: React.FC<Props> = ({ status, isFetching, data }) => {
+  // if the data is not loaded, show loader
+  if (status === "loading" || !data) {
+    return <div>Loading</div>;
+  }
+
+  // I need to start using something else over console logs
+  // asked some programmer friends what tools they use
+  // What do you use Mikkel?
+  // console.log(`status: ${status}`);
```

Comment:
I have nothing against console.log. But it is also possible to setup vs code to use step debugging. Maybe we could look at that together, one of these days?
Anyways, I think we should remove these comments since they don't add value to the code base :)

Reply:
Yes I'd like that very much.
And removinization is eminent :)